pkg/util/validation/errors.go

```diff
@@ -155,29 +183,40 @@ func (e *exemplarValidationError) Error() string {
 	return fmt.Sprintf(e.message, e.timestamp, mimirpb.FromLabelAdaptersToLabels(e.seriesLabels).String(), mimirpb.FromLabelAdaptersToLabels(e.exemplarLabels).String())
 }
 
-func newExemplarEmtpyLabelsError(seriesLabels []mimirpb.LabelAdapter, exemplarLabels []mimirpb.LabelAdapter, timestamp int64) ValidationError {
+var exemplarEmptyLabelsMsgFormat = globalerrors.Format(
+	globalerrors.ErrIDExemplarLabelsMissing,
+	"received exemplar with no valid labels, timestamp: %%d series: %%s labels: %%s")
+
+func newExemplarEmptyLabelsError(seriesLabels []mimirpb.LabelAdapter, exemplarLabels []mimirpb.LabelAdapter, timestamp int64) ValidationError {
 	return &exemplarValidationError{
-		message:        "exemplar missing labels, timestamp: %d series: %s labels: %s",
+		message:        exemplarEmptyLabelsMsgFormat,
 		seriesLabels:   seriesLabels,
 		exemplarLabels: exemplarLabels,
 		timestamp:      timestamp,
 	}
 }
 
+var exemplarMissingTimestampMsgFormat = globalerrors.Format(
+	globalerrors.ErrIDExemplarTimestampInvalid,
+	"received exemplar with no timestamp, timestamp: %%d series: %%s labels: %%s")
+
 func newExemplarMissingTimestampError(seriesLabels []mimirpb.LabelAdapter, exemplarLabels []mimirpb.LabelAdapter, timestamp int64) ValidationError {
 	return &exemplarValidationError{
-		message:        "exemplar missing timestamp, timestamp: %d series: %s labels: %s",
+		message:        exemplarMissingTimestampMsgFormat,
 		seriesLabels:   seriesLabels,
 		exemplarLabels: exemplarLabels,
 		timestamp:      timestamp,
 	}
 }
 
-var labelLenMsg = "exemplar combined labelset exceeds " + strconv.Itoa(ExemplarMaxLabelSetLength) + " characters, timestamp: %d series: %s labels: %s"
+var exemplarMaxLabelLengthMsgFormat = globalerrors.Format(
+	globalerrors.ErrIDExemplarLabelsTooLong,
+	"received exemplar whose combined labels set size exceeds the limit of %d characters, timestamp: %%d series: %%s labels: %%s",
```

Comment:
```suggestion
	"received an exemplar where the set size of its combined labels exceeds the limit of %d characters, timestamp: %%d series: %%s labels: %%s",
```

Reply:
> where the set size

Is "set size" intentional here or a leftover from the original one?

Reply:
Only to the extent that I am trying to nail down the meaning of the initial change. :)

Based on your feedback, _set_ is a past participle. Adjusting my suggestion...

Reply:
Is it clear if we remove "set"?

Reply:
Yes, removing the "set" LGTM!